ams/server/src/main/java/com/netease/arctic/server/persistence/mapper/OptimizingMapper.java

```diff
@@ -110,20 +114,22 @@ List<TableOptimizingProcess> selectSuccessOptimizingProcesses(
           "end_time, status, fail_reason, optimizer_token, thread_id, rewrite_output, metrics_summary) VALUES ",
       "<foreach collection='taskRuntimes' item='taskRuntime' index='index' separator=','>",
       "(#{taskRuntime.taskId.processId}, #{taskRuntime.taskId.taskId}, #{taskRuntime.retry}," +
-          " #{taskRuntime.tableId}, #{taskRuntime.partition}, #{taskRuntime.startTime}," +
-          " #{taskRuntime.endTime}, #{taskRuntime.status}, #{taskRuntime.failReason}," +
-          " #{taskRuntime.optimizingThread.token}, #{taskRuntime.optimizingThread.threadId}," +
-          " #{taskRuntime.outputBytes," +
-          " typeHandler=com.netease.arctic.server.persistence.converter.JsonSummaryConverter}," +
+          " #{taskRuntime.tableId}, #{taskRuntime.partition}, #{taskRuntime.startTime, typeHandler=com.netease.arctic" +
```

Comment:
I suggest adjusting the line breaks to ensure that the content of the typeHandler appears completely on one line.